utils/build.ps1

```diff
@@ -2293,11 +2293,14 @@ function Write-SDKSettingsPlist([OS] $OS) {
   Write-PList -Settings $SDKSettings -Path "$(Get-SwiftSDK $OS)\SDKSettings.plist"
 
   $SDKSettings = @{
-    CanonicalName = $OS.ToString()
-    DisplayName = $OS.ToString()
-    IsBaseSDK = "NO"
+    CanonicalName = $Platform.ToString()
+    DisplayName = $Platform.ToString()
+    IsBaseSDK = "YES"
     Version = "${ProductVersion}"
     VersionMap = @{}
+    HeaderSearchPaths = @( "usr/include" );
+    LibrarySearchPatsh = @();
```

Comment:
typo: LibrarySearchPatsh -> LibrarySearchPaths, but this could also be dropped since it's optional

Reply:
Need to see if we need any additional library search paths. Having that there "documented" for the future seems nice.